.travis.yml

```diff
@@ -4,9 +4,10 @@ before_script:
  - chmod +x build.sh
 
 script: 
-  - ./build.sh RunTests
+  - ./build.sh GenerateDocs
```

Comment:
Can you move this change to a separate PR please. I will be happy to accept it but I'd like it in a separate commit.